### DISCUSSION TITLE
Tool List?

### DISCUSSION THREAD

**Discussion by zer0uid** [ThePorgs/Exegol#90] [Q&A]
First and foremost, great work; enjoying it!  Is there a list of tools that are currently installed?

**Comment by ShutdownRepo** [comment]
Hey there 👋 thank you for you feedback! At the beginning of the project, there was a tool list, but it was never updated, and it was removed for release 4.0.0. However, for the upcoming release, there are a few things we'd like to work on, documentation being on of those, and it includes a tools list.

**Reply by zer0uid** [reply]
If there's anything I can assist with regarding documentation or the project; please let me know.

**Reply by ShutdownRepo** [reply]
Definitely! There's a ton of work we have on our plates right now, either technical or documentation-wise. Feel free to DM me on Twitter ([_nwodtuhs](https://twitter.com/_nwodtuhs)) and we'll identify next steps 😉
